pkg/api/groups.go

```diff
@@ -694,7 +720,17 @@ func (api *API) GetGroupVersionCountTimeline(groupID string, duration string) (m
 		}
 	}
 
-	return timelineCount, nil
+	go func() {
+		cachedGroupVersionCountLock.Lock()
+		defer cachedGroupVersionCountLock.Unlock()
+		val, ok := cachedGroupVersionCount[cacheKey]
+		if !ok || time.Since(val.storedAt) > cachedGroupVersionCountLifespan {
```

Comment:
The data in cache is valid if time since it was updated is less than a minute ago. Thus data in cache is stale if it's one minute or more. So:
```suggestion
		if !ok || time.Since(val.storedAt) >= cachedGroupVersionCountLifespan {
```